src/interaction.service.ts

```diff
@@ -7,17 +7,17 @@ const endpoint = 'locations';
 
 export default (axios: AxiosInstance) => {
   return {
-    getInteractions:(locationId:string, params: InteractionParams) => {
+    getInteractions:(locationId:string, params?: InteractionParams) => {
       return axios.get(`${endpoint}/${locationId}/reviews`, {params});
     },
-    getInteractionById:(params:Interactions)=>{
-      return axios.get(`${endpoint}/reviewDetails`,{params});
+    getInteractionById:(intractionByIds?:Array<string>)=>{
+      return axios.get(`${endpoint}/reviewDetails${intractionByIds}`);
```

Comment:
this is incomplete, should be `${endpoint}/reviewDetails?interactionIds=${intractionByIds}`